style.css

```diff
@@ -0,0 +1,859 @@
+    padding-bottom: 30px;
+  }
+
+  .text1 {
+    color: #000;
+    width: 250px;
+    height: auto;
+  }
+
+  .text2 {
+    color: chocolate;
+    width: 250px;
+    height: auto;
+  }
+
+  .text3 {
+    color: #272a31;
+    width: 250px;
+    height: auto;
+  }
+
+  .speakers {
+    display: grid;
+    grid-template-columns: 200px 200px;
+    grid-template-rows: repeat(3, 120px);
+    row-gap: 0;
+    column-gap: 150px;
+    margin-left: 20%;
+    justify-content: left;
+    margin-top: 0;
+    width: 100%;
+  }
+
+  .title-Main {
+    position: absolute;
+    top: 5px;
+    left: 45%;
+    color: hsl(223, 16%, 91%);
+    font-family: 'Lato', sans-serif;
+    line-height: 1.5;
+    font-size: large;
+  }
+
+  .title-Main2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    color: hsl(222, 11%, 17%);
+    font-family: 'Lato', sans-serif;
+    line-height: 1.5;
+    font-size: large;
+  }
+
+  /* ################################------start---block speaker person----------------##################################################### */
+  .speaker-person1 {
+    display: flex;
+    line-height: 1.5;
+    font-size: 14px;
+  }
+
+  .speaker-person2 {
+    display: flex;
+    line-height: 1.5;
+    font-size: 14px;
+  }
+
+  .speaker-person3 {
+    display: flex;
+    line-height: 1.5;
+    font-size: 14px;
+  }
+
+  .speaker-person4 {
+    display: flex;
+    line-height: 1.5;
+    font-size: 14px;
+  }
+
+  .speaker-person5 {
+    display: flex;
+    line-height: 1.5;
+    font-size: 14px;
+  }
+
+  .speaker-person6 {
+    display: flex;
+    line-height: 1.5;
+    font-size: 14px;
+  }
+
+  /* ################################------end---block speaker person----------------##################################################### */
+
+  .imgP1 {
+    width: 80px;
+    height: 80px;
+    padding-right: 10px;
+  }
+
+  .imgP2 {
+    width: 100px;
+    height: 100px;
+    padding-right: 10px;
+  }
+
+  .imgP3 {
+    width: 100px;
+    height: 100px;
+    padding-right: 10px;
+  }
+
+  .imgP4 {
+    width: 100px;
+    height: 100px;
+    padding-right: 10px;
+  }
+
+  .imgP5 {
+    width: 100px;
+    height: 100px;
+    padding-right: 10px;
+  }
+
+  .imgP6 {
+    width: 100px;
+    height: 100px;
+    padding-right: 10px;
+  }
+
+  .footer {
+    position: relative;
+    height: 60vh;
+    width: auto;
+  }
+
+  .footer1 {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    height: 30vh;
+    width: 100%;
+    background-color: #272a31;
+  }
+
+  .footer1 .footer11 {
+    display: flex;
+    margin-top: 20px;
+    justify-content: center;
+    color: #d3d3d3;
+    width: 100%;
+  }
+
+  .footer12 {
+    position: relative;
+    width: 60px;
+    height: 1px;
+    left: calc(50% - 60px / 2 - 0.5px);
+    border-radius: 10px;
+    background-color: hsl(6, 82%, 59%);
+    margin-bottom: 20px;
+  }
+
+  .footer13 {
+    display: flex;
+    justify-content: space-around;
+    margin-bottom: -45px;
+  }
```

Comment:
- [ ] Kindly use the given `color` for the `contents/lists` of this section, and also decrease the space between them to match the design guideline.

- Given Design:
- ![image](https://user-images.githubusercontent.com/60083437/227037007-45d944ce-5ae7-475a-a571-ddf6539c1e8d.png)